### DISCUSSION TITLE
Make Japa Test Runner Optional when creating new apps

### DISCUSSION THREAD

**Discussion by mikesaintsg** [adonisjs/core#3709] [General]
It would be great if the japa test runner were optional like eslint, prettier, and webpack encore are since there's so much boilerplate that isn't needed when using our own test runner. I really like the modularity of Adonisjs. Right now, it feels a bit too opinionated to have it not ne an option. Please and thank you for all that you do!

**Comment by RomainLanz** [comment]
Hey @mikesaintsg! 👋🏻 

Maybe @Julien-R44 can make the change in https://github.com/adonisjs-community/create-adonis-ts-app/issues/43

**Reply by Julien-R44** [reply]
I've just made a PR : https://github.com/adonisjs-community/create-adonis-ts-app/pull/44 ! 😀

**Reply by mikesaintsg** [reply]
Thanks guys, much appreciated!
